src/main/java/org/havenapp/main/ListActivity.java

```diff
@@ -78,16 +74,21 @@
 
     private Handler handler = new Handler();
 
+    @SuppressLint("SimpleDateFormat")
```

Comment:
Wouldn't it be better to use `new SimpleDateFormat("dd MMM yyyy", Locale.US)` instead of suppresing lint?

Reply:
yeah that would be better.will change it